proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -202,3 +221,24 @@ message DeleteRepositoryByFullNameRequest {
 }
 
 message DeleteRepositoryByFullNameResponse {}
+
+message DeprecateRepositoryByNameRequest{
+  string owner_name = 1;
+  string repository_name = 2;
+  // A message shown along with the deprecation warning for this repository.
+  // It must be a utf8 encoded string containing 256 or fewer characters.
```

Comment:
Comment is fine as-is, but just so you know the `utf-8` aspect is guaranteed by the `string` definition in Protobuf. You can see the distinction between `string` and `bytes` here: https://developers.google.com/protocol-buffers/docs/proto3#scalar